hana/install.sls

```diff
@@ -6,7 +6,7 @@
 
 hana_install_{{ node.host+node.sid }}:
   hana.installed:
-    - sid: {{ node.sid }}
+    - name: {{ node.sid }}
```

Comment:
@arbulu89  is this change to name intentional? sid looks more appropriate to me...

Reply:
@diegoakechi Yes, it is intentional. Due some changes in the last salt version (fluorine), name argument must be used as a first parameter. Otherwise an error is raise. 
I agree that **sid** would be more appropriate, but in order to make everything work in different salt version we are forced to update this (in fact, all of the methods of other salt states code have **name** as first parameter)

Reply:
@arbulu89 hum.. ok, sad, but I understand now... hehe.